src/Query/QueryBuilder.php

```diff
@@ -81,6 +81,14 @@ public function setMinimumScore($minScore): QueryBuilder
         return $this;
     }
 
+    public function addSort($field, $order): QueryBuilder
```

Comment:
IMO: as the method is implemented by the derived class itself, I think it's better to use `self`